### PR TITLE
Add liveness-job for mongodb

### PR DESCRIPTION
**Pull request by shashank855**
This commit does the following:
1. Runs liveness-job(mongo_liveness.yml) for checking the liveness of mongodb.
2. Python-script(server.py) which runs as a part of liveness-job to fetch the status of mongodb.

Signed-off-by: ranjanshashank <ranjanshashank855@gmail.com>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
Liveness running when database is healthy/ connection is established 
```
kubectl logs -f livess-mongo-89cpc
Database(MongoClient(host=['mongo.litmus.svc.cluster.local:27017'], document_class=dict, tz_aware=False, connect=True), 'admin')
{'available': 838858, 'current': 2, 'totalCreated': 2}
pass 1
liveness Running
liveness Running
liveness Running
liveness Running
liveness Running
liveness Running
liveness Running
liveness Running
```

### CODE REVIEW

File: apps/mongodb/liveness/mongo_liveness.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: livess-mongo
```
Comment (ksatchit) [inline apps/mongodb/liveness/mongo_liveness.yml]: typo ??

**Comment by ksatchit** [comment]
@shashank855 , let us add the ENV to take the mongo fqdn

File: apps/mongodb/liveness/mongo_liveness.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+spec:
+  template:
+    metadata:
+      name: liveness
+   
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: liveness
+        image: ranjnshashank855/py-server:t2
```
Comment (ksatchit) [inline apps/mongodb/liveness/mongo_liveness.yml]: can we have this image in the openebs repo ?

Comment (shashank855) [reply apps/mongodb/liveness/mongo_liveness.yml]: @ksatchit Sure sir

**Pull request merged**